packages/controllers/organization_detail.coffee

```diff
@@ -5,7 +5,60 @@ if Meteor.isClient
   Template.organizationDetail.helpers
     organization: ->
       Organizations.findOne(@organizationId)
+    userIsMember: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId(),
+        memberOfOrgs: @organizationId
+      })
+    userIsAdmin: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId()},
+        $in: {adminOfOrgs: [@organizationId]}
+      )
+    hideButtonsForProfile: (profileId) ->
```

Comment:
is this to make it so admins can't remove themselves? why shouldn't they, unless they're the only admin?